composite/action.yml

```diff
@@ -175,12 +175,13 @@ runs:
         esac
       shell: bash
 
-    - name: Cache PIP Packages
-      uses: actions/cache@v3
+    - name: Restore PIP packages cache
+      uses: actions/cache/restore@v3
+      id: cache
       continue-on-error: true
       with:
         path: ${{ steps.os.outputs.pip-cache }}
-        key: enricomi-publish-action-${{ runner.os }}-${{ runner.arch }}-pip-${{ steps.python.outputs.version }}-${{ hashFiles('**/requirements.txt', 'composite/action.yml') }}
+        key: enricomi-publish-action-${{ runner.os }}-${{ runner.arch }}-pip-${{ steps.python.outputs.version }}-f3f2c295046c91ed612b4efb6c9fb352
```

Comment:
I saw you were fighting with this, in the end you'll update it manually for each release?

Reply:
it's in `glob`: https://github.com/actions/toolkit/blob/main/packages/glob/src/glob.ts#L29

Reply:
> you'll update it manually for each release

yes, but only if `requirements.txt` changes

Reply:
> and call `hashFiles` directly from JS?

maybe, but I'd prefer not to add more complexity